core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -1236,17 +1236,20 @@ public void testWeekOfYearWithTimeType() {
                 DSL.week(
                     functionProperties, DSL.literal(new ExprTimeValue("12:23:34")), DSL.literal(0)),
                 "week(TIME '12:23:34', 0)",
-                LocalDate.now(functionProperties.getQueryStartClock()).get(ALIGNED_WEEK_OF_YEAR)),
+                LocalDate.now(functionProperties.getQueryStartClock()).get(ALIGNED_WEEK_OF_YEAR)
```

Comment:
ignore this test case and add issue link? my concern is this temporary fix could fail again.

Reply:
Tied ignoring the coverage is falling below 100% percent.
It won't fail again this year. 2024 Jan 1 is Monday as a result week of the year is off by 1.